time-tracker/src/reducers.jsx

```diff
@@ -0,0 +1,61 @@
+import { createAction, createReducer } from "@reduxjs/toolkit";
+
+export const changeCurrentTask = createAction("changeCurrentTask");
+export const pauseAllTasks = createAction("pauseAllTasks");
+export const addNewTask = createAction(
+  "addNewTask",
+  (taskName = "TASK NAME NOT SET") => {
```

Comment:
In a real application you would probably write something more useful from the get go, like "New task"